Content.Server/Storage/Components/EntityStorageComponent.cs

```diff
@@ -1,97 +1,12 @@
 using Content.Server.Atmos;
-using Content.Shared.Physics;
-using Content.Shared.Whitelist;
-using Robust.Shared.Audio;
-using Robust.Shared.Containers;
+using Content.Shared.Storage.Components;
+using Robust.Shared.GameStates;
 
 namespace Content.Server.Storage.Components;
 
-[RegisterComponent]
-public sealed class EntityStorageComponent : Component, IGasMixtureHolder
+[RegisterComponent, NetworkedComponent, ComponentReference(typeof(SharedEntityStorageComponent))]
+public sealed class EntityStorageComponent : SharedEntityStorageComponent, IGasMixtureHolder
 {
```

Comment:
You can put gasmixtures on shared I thought, also sidenote networkedcomponent is inheritable

Reply:
nope, atmos mixtures are server-only.